docs/vexim-acl-check-rcpt.conf

```diff
@@ -52,5 +52,16 @@
 #                          and spamassassin='1'}}}{1} {yes}{no}}
 #          !acl          = spf_rcpt_acl
 
+#
+# To use a DNSBL the best would be to set up a caching dns server for you own
+# otherwise you might get rate limited and your requests are not processed
+# 
+# Also disable spamchecking for authenticated users because dial up networks
```

Comment:
Can you reword this so that it doesn't sound like we're actually disabling spam checks here, but just DNSBL checks?

Also, I'm unsure if the comment above is even necessary, but even if you want it to stay, I don't think the second one should start with the word "Also".